app/views/user/CertificatesView.coffee

```diff
@@ -55,6 +58,9 @@ module.exports = class CertificatesView extends RootView
     @supermodel.trackRequest @courseLevels.fetchForClassroomAndCourse classroomID, courseID, data: { project: 'concepts,practice,assessment,primerLanguage,type,slug,name,original,description,shareable,i18n,thangs.id,thangs.components.config.programmableMethods' }
     @listenToOnce @courseLevels, 'sync', @calculateStats
 
+    if features.china
+      @serialNumber = @hashString(@user.id, @courseInstanceID)
```

Comment:
This will just hash the @user.id parameter, because the `hashString` method only looks at the first argument. Should concatenate the two so that the same user gets different certificate serial numbers for different courses:

```suggestion
      @serialNumber = @hashString(@user.id + @courseInstanceID)
```